src/features/agenda/queries/agendaHooks.ts

```diff
@@ -330,10 +330,22 @@ export const useGetAgendaWeeks = (mondays: DateTime[]) => {
     return queries.some(q => q.isLoading);
   }, [mondays, queries]);
 
+  const refetch = async () => {
```

Comment:
making this function async is "dangerous" as there's nothing awaited here. either you return a promise all, or consider it not async